include/module.hpp

```diff
@@ -12,13 +12,20 @@ struct Module {
     Module(std::string name) : name(name) {}
 };
 
+struct ModuleSymbol {
+    virtual void InjectSymbols(sol::state& lua) = 0;
```

Comment:
To może być rozwiązane statycznie. Nie chcemy dwóch poziomów VTable'owania, sugerował bym zrobić to nievirtualnie.

Reply:
tymczasowo zmienione, czekam na propozycje

Reply:
zmergowane poprawki